pkg/constant/constants.go

```diff
@@ -9,16 +9,33 @@ const (
 	PorterAnnotationKey   string = "lb.kubesphere.io/v1alpha1"
 	PorterAnnotationValue string = "porter"
 
-	//Indicates the node to which layer2 traffic is sent
+	// Indicates the node to which layer2 traffic is sent
 	PorterLayer2Annotation string = "layer2.porter.kubesphere.io/v1alpha1"
 
+	PorterLBSAnnotationKey     string = "lbs.porter.kubesphere.io/v1alpha1"
+	PorterOnePort              string = "one"
+	PorterAllNode              string = "all"
+	PorterNodeExtnlIPPrefLabel string = "lbs.porter.kubesphere.io/external-ip-preffered"
+	PorterNodeExcludeLBSLabel  string = "lbs.porter.kubesphere.io/exclude.node"
+	PorterLBSExposedExternalIP string = "lbs.porter.kubesphere.io/exposed.node.external-ips"
+	PorterLBSExposedInternalIP string = "lbs.porter.kubesphere.io/exposed.node.internal-ips"
+	PorterForwardImage         string = "kubespheredev/openelb-forward:v0.4.2"
+	PorterProxyImage           string = "kubespheredev/openelb-proxy:v0.4.2"
+	NameSeparator              string = "-"
+	IPSeparator                string = ","
+	EnvArgSplitter             string = " "
+	PorterDeDsPrefix           string = "svc-proxy-"
+	PorterLBSFInalizer         string = "lbs.porter.kubesphere.io/finalizer"
```

Comment:
How about adding a protocol called `node-proxy`? Use the annotation protocol.porter.kubesphere.io/v1alpha1: node-proxy to enable this mode. 
In this case, the code here could be redefined with the following changes
```suggestion
	NodeProxyTypeAnnotationKey        string = "node-proxy.porter.kubesphere.io/type"
	NodeProxyTypeDeployment           string = "deployment"
	NodeProxyTypeDaemonSet            string = "daemonset"
	LabelNodeProxyExternalIPPreffered string = "node-proxy.porter.kubesphere.io/external-ip-preffered"
	LabelNodeProxyExcludeNode         string = "node-proxy.porter.kubesphere.io/exclude-node"
	NodeProxyExternalIPAnnotationKey  string = "node-proxy.porter.kubesphere.io/external-ip"
	NodeProxyInternalIPAnnotationKey  string = "node-proxy.porter.kubesphere.io/internal-ip"
	NodeProxyForwardImage             string = "kubespheredev/openelb-forward:v0.4.2"
	NodeProxyProxyImage               string = "kubespheredev/openelb-proxy:v0.4.2"
	NameSeparator                     string = "-"
	IPSeparator                       string = ","
	EnvArgSplitter                    string = " "
	NodeProxyWorkloadPrefix           string = "node-proxy-"
	NodeProxyFinalizerName            string = "node-proxy.porter.kubesphere.io/finalizer"
```